PacketDotNet/LLDP/ManagementAddress.cs

```
@@ -291,8 +291,10 @@ public string ObjectIdentifier
         {
             get
             {
-                return UTF8Encoding.UTF8.GetString(tlvData.Bytes, ObjectIdentifierOffset,
+                if (ObjIdLength > 0)
+                    return UTF8Encoding.UTF8.GetString(tlvData.Bytes, ObjectIdentifierOffset,
                                             ObjIdLength);
+                else return "0";
```

does it make sense to fail silently like this and return "0" vs. throwing?